src/components/Card.jsx

```diff
@@ -1,64 +1,69 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import { connect } from 'react-redux'
 import { likeOrDislike, toggleSignInModal } from '../actions/actionCreators'
 
 const whiteBackground = {
   backgroundColor: '#FFF'
-}
+};
 
-class Card extends React.Component {
-  state = {
-    liked: false
-  }
+const Card = ({
+  props: {
```

Comment:
Sorry, I get an error here saying props is not defined. Can we have the destructuring without it e.g
`const Card = ({ dispatch, home: { loggedIn }, index, project })  => {`